File: 백준/Bronze/2501. 약수 구하기/약수 구하기.c
#include <stdio.h>

int main(void) {
	int arr[10000] = { 0,  };
	int n, k, cnt = 0;
	scanf("%d %d", &n, &k);
	for (int i = 1; i < n + 1; i++) {
		if (n % i == 0) {
			arr[cnt] = i;
			cnt++;
		}
	}
	if (cnt < k) printf("%d", 0);
	else printf("%d", arr[k - 1]);
							 
	return 0;				 
}	

File: 백준/Bronze/2501. 약수 구하기/약수 구하기.py
arr = []
n, k = map(int, input().split())
cnt = 0
for i in range(1, n + 1):
    if(n % i == 0):
        arr.append(i)
        cnt += 1
if(cnt < k):
    print("0")
else:
    print("{}".format(arr[k - 1]))